.ci/run_metrics_ci.sh

```diff
@@ -125,18 +125,8 @@ pushd "$CURRENTDIR/../metrics"
 	# ops/second
 	bash network/network-nginx-ab-benchmark.sh
 
-	# ping latency
-	bash network/network-latency.sh
```

Comment:
I see that we still run these tests in `metrics/run_all_metrics.sh`, but I'm now wondering if we shouldn't just have a single parameterised script to handle all metrics tests?

Reply:
as in a single script where we can say 'script --all' or 'script --kpis' for instance, to remove the duplication between the two scripts?

We could do that. In my mind we'd re-factor the run_all_metrics (well, and change the name). We'd have to add the KSM handling and some other trickery from the run_metrics_ci script as well (like the results storing options as well).

How about I open an Issue to note we could do with a refactor of the scripts?

Reply:
Thanks - Sounds good to me.

Reply:
See #912.